waifus/kyoko kirigirl.json

```diff
@@ -0,0 +1,14 @@
+{
+  "type": 0,
+  "english-name": "Kyoko Kirigiri",
+  "japanese-name": "霧切 響子",
+  "aliases": ["Ultimate ???", "Super High School Level ???"],
+  "image": "https://vignette.wikia.nocookie.net/danganronpa/images/1/12/Kyoko_Kirigiri_Illustration.png",
+  "month": 6,
+  "day-of-month": 10,
```

Comment:
This is incorrect, I think you meant to put the 6 on ``day-of-month`` and ``month`` with 10

Reply:
You're right - nice catch. Also, I just noticed that I made a typo in the file name.